### PR TITLE
Hello microverse project

### PR DESCRIPTION
**Pull request by DhuruvSingh**
## Hello Microverse project

- Created a new GitHub repo.
- Created a new branch according
- Added a .gitignore file.
- Set up a linter for HTML and CSS
- Added a descriptive README file and customized it
- Added 3 files (each one in a separate commit):
  1.   A .html file that includes a header with the text "Hello Microverse!". Use h1 tag for that.
  2.   A .css file that includes one class with styles for your header.
  3.   A test.md file with the text "This file should be ignored by git".

> Please review and give your valuable feedback

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,62 @@
...
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
```
Comment (omarsalem7) [inline README.md]: - [x] Kindly remove this to clean your readme and add instructions for how to clone your repo.

File: test.md [hunk 2]
```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```
Comment (omarsalem7) [inline test.md]: - [x] This file should be ignored and not appear in your remote repo. You can take a look in [this link](https://stackoverflow.com/questions/7927230/remove-directory-from-remote-repository-after-adding-them-to-gitignore) to help you how to remove files from the remote repo after push.

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @DhuruvSingh 😊, 

Good job so far! 👏
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

## Highlights
- Descriptive pull request ✔️
- No linter errors ✔️
- Descriptive commit messages ✔️

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sumairq — APPROVED** [review]
### Hi @DhuruvSingh , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Github flow

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by DhuruvSingh** [comment]
Thank You so much for your valuable feedback @omarsalem7 & @sumairq

**Pull request merged**